### DISCUSSION TITLE
(PC) Left button not responding

### DISCUSSION THREAD

**Discussion by Floffidinio** [ppy/osu#17642] [Q&A] [answered]
### Type

Game behaviour

### Bug description

Left button or for me z didn't respond when I pressed it.
It kinda worked the first time I opened Osu! Lazer. Sometimes it responded and sometimes not but I thought it was my PC so I just quit the game (did not restart/shut down the pc). Tried again a few hours later and did not respond at all. Fixed when I went to input and changed the left button from z to z again.

### Version

2022.327.0-lazer

### Logs

[database.log](https://github.com/ppy/osu/files/8405571/database.log)
[network.log](https://github.com/ppy/osu/files/8405572/network.log)
[performance.log](https://github.com/ppy/osu/files/8405573/performance.log)
[runtime.log](https://github.com/ppy/osu/files/8405574/runtime.log)
[updater.log](https://github.com/ppy/osu/files/8405575/updater.log)

**Comment by Floffidinio** [comment]
Update: changing left button input from z to z _did not_ fix the problem. It worked for 2 beatmaps and after that, it stopped working again mid-game

**Comment by smoogipoo** [comment]
Sounds like the game is being unfocused, or some background app is stealing your input.

**Reply by peppy** [reply]
Likely, would want to see a video to figure out what's going on.

**Comment by bdach** [comment, marked as answer]
if you're using a tablet to play the game then please try disabling mouse button in gameplay. you may be pressing the stylus into the tablet digitizer, causing clicks.

**Reply by Floffidinio** [reply]
Thank you. Just restarted my pc with everything deleted so all my osu settings were back to default. Thought I did everything but it looks like I forgot this. Man I feel stupid right now
